tests/test_motiondetector.py

```diff
@@ -86,7 +88,29 @@ def test_empty_config_name(self):
         md = getMD()
         raw_config = {"cam_name": ""}
         config = make_component_config(raw_config)
-        with pytest.raises(ValueError, match="Source camera must be provided as 'cam_name'"):
+        with pytest.raises(ValueError, match=pytest.source_camera_name_none_defined_error_message):
+            response = md.validate_config(config=config)
+
+    # For each way to specify a valid camera name, test that the return is valid.
+    @parameterized.expand((
+        ("cam_name defined, camera_name not defined",  {"cam_name": "test"}),
+        ("camera_name defined, cam_name not defined",  {"camera_name": "test"}),
+    ))
+    def test_valid_camera_names(self, unused_test_name, cam_config):
+        md = getMD()
+        config = make_component_config(cam_config)
+        response = md.validate_config(config=config)
+        assert response == ["test"]
+
+    # For each way to spedify an invalid camera name, test that the expected error is raised.
+    @parameterized.expand((
+        ("cam_name not defined, camera_name not defined", {}, pytest.source_camera_name_none_defined_error_message),
```

Comment:
This is super nitpicky but add another one of these that has {"camera_name": "", "cam_name": ""}. Should definitely spit out the none_defined_error_thingy but it's nice to make sure (and it's not _exactly_ the same as the empty one).